mev-build-rs/src/utils/compat.rs

```diff
@@ -64,3 +65,27 @@ pub fn to_execution_payload(value: &SealedBlock) -> ExecutionPayload {
     };
     ExecutionPayload::Deneb(payload)
 }
+
+pub fn to_blobs_bundle(sidecars: &[BlobTransactionSidecar]) -> BlobsBundle {
+    let mut commitments = List::default();
+    let mut proofs = List::default();
+    let mut blobs = List::default();
+
+    // TODO: perform length checks to avoid panic on `push`
+    for sidecar in sidecars {
+        for commitment in &sidecar.commitments {
```

Comment:
these nested loops are not ideal, but in the usual case should not add much overhead.

it would be nice if there was an infallible conversion from the `FixedBytes<N>` to the `ByteVector<N>`.